### DISCUSSION TITLE
Using with Vue test utils

### DISCUSSION THREAD

**Discussion by DevCJames** [vuejs/language-tools#114] [Q&A] [answered]
Was just wondering if its possible to get this working with vue test utils and or jest?
At the min I can get the correct typings while writing tests, however once I run my test I get typing errors.

Would there be a way around this or a fix? Or is it out of scope for what the extension can do?

**Comment by johnsoncodehk** [comment]
You may need to note that problem: https://github.com/johnsoncodehk/volar/issues/94

> however once I run my test I get typing errors.

What errors you get?

**Reply by DevCJames** [reply]
So for example calling
```
expect(wrapper.vm.testMethod()).toEqual('hello')
```
Works fine in vscode. I have the proper typings etc, however running the test I just get 
```
Property 'testMethod' does not exist on type 'CombinedVueInstance<Vue, object, object, object, Record<never, any>>'.
```
for any variables or methods.

**Reply by johnsoncodehk** [reply]
It's seem a Vue 2 types problem, I can't check it without a repro.

You can upload a repro, or ask to https://discord.com/channels/325477692906536972/331999220125138944.

**Comment by ghost** [comment, marked as answer]
For Vue 3, see https://github.com/vuejs/vue-test-utils/issues/255#issuecomment-956079773
